packages/lab/src/avatar/Avatar.css

```diff
@@ -2,65 +2,36 @@
 .saltAvatar {
   --avatar-background: var(--saltAvatar-background, var(--salt-accent-background));
   --avatar-foreground: var(--saltAvatar-foreground, var(--salt-accent-foreground));
```

Comment:
I'd say to move these to below directly in the attribute, no need for a local token since it's only being used in 1 place

Reply:
as in just have `background: var(--saltAvatar-background, var(--salt-accent-background));` below, + same for foreground